pkg/scheduler/plugins/priority/priority.go

```diff
@@ -74,6 +164,82 @@ func (pp *priorityPlugin) OnSessionOpen(ssn *framework.Session) {
 	}
 
 	ssn.AddJobOrderFn(pp.Name(), jobOrderFn)
+
+	priorityFn := func(task *api.TaskInfo, node *api.NodeInfo, nodes []*api.NodeInfo) (int, error) {
+		nodeInfo := cache.NewNodeInfo(node.Pods()...)
+		nodeInfo.SetNode(node.Node)
+		var score int = 0
+
+		host, err := priority.ImageLocalityPriorityMap(task.Pod, len(nodes), nodeInfo)
```

Comment:
only include "image locality", "LeastRequestedPriorityMap" and "pod affinity" should be enough.

Reply:
and "node affinity".